editor.js

```diff
@@ -3,7 +3,7 @@
 	Extension Editor for XKit 7
 	Version 1.0
 
-	(c) 2011 - 2013 STUDIOXENIX
+	(c) 2011 - 2015 STUDIOXENIX & New-XKit Team
```

Comment:
@0xazure [IANAL] as I understand it, each contributor individually licenses
their contributions under the gplv3, but retains their own copyright. this
leads to tricky situations wrt relicensing and dual licensing, and stuff
like that, which is why CLA are a thing. But you need some sort of legal
entity to take ownership to make a CLA work, and i don't really think its
very useful to us. (are we ever planning on relicensing away to a
non-compatible license?

On Wed, Sep 2, 2015 at 12:08 AM 0xazure <notifications@github.com> wrote:

> In editor.js
> <https://github.com/new-xkit/XKit/pull/405#discussion_r38497432>:
>
> > @@ -3,7 +3,7 @@
> >  	Extension Editor for XKit 7
> >  	Version 1.0
> >
> > -	(c) 2011 - 2013 STUDIOXENIX
> > +	(c) 2011 - 2015 STUDIOXENIX & New-XKit Team
>
> can't really be thought of as such from a copyright point of view
>
> Are we not releasing the software as a group ("New XKit") under the GPLv3?
> We don't have an official Contributor Agreement that our contributors sign
> before releasing their code to us, but at this point it's kind of implied,
> no?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/405/files#r38497432>.
>